### PR TITLE
Vue 3 support

### PR DESCRIPTION
**Pull request by salmi3d**
Added support for Vue 3 API

### CODE REVIEW

**Comment by chenguzhen87** [comment]
```  onChange: this.$listeners ? this.$listeners.onChange : this.$attrs.onOnChange
```
改为 onChange: this.$listeners ? this.$listeners.onChange : this.$attrs.onChange ?
用例增加  <GesturePassword :width='width' :height='height'  :onChange="onChange" /> ?

**Comment by salmi3d** [comment]
Nope. There is no more $listeners in Vue 3 API. All component listeners now in $attr

**Comment by chenguzhen87** [comment]
@salmi3d 
··· js
 'this.$attrs.onOnChange'  can be changed to 'this.$attrs.onChange' ? 
```
Thanks for your reply

File: packages/gesture-password-vue/src/gesture-password.vue [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default {
           ...props,
           width: px2hd(props.width),
           height: px2hd(props.height),
-          onChange: this.$listeners.onChange
+          onChange: this.$listeners ? this.$listeners.onChange : this.$attrs.onOnChange
```
Comment (xiaohuoni) [inline packages/gesture-password-vue/src/gesture-password.vue:60]: this.$listeners ? this.$listeners.onChange : this.$attrs.onChange

**Comment by salmi3d** [comment]
All properties of $attrs that starts with on... are listeners. So either
this.$listeners ? this.$listeners.onChange : this.$attrs.onOnChange 
+
<GesturePassword :width="width" :height='height' @onChange="onChange" />
or
this.$listeners ? this.$listeners.change : this.$attrs.onChange
+
<GesturePassword :width="width" :height='height' @change="onChange" />

**Pull request merged**